db/migrations/20190919123945_create_movies_locations.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 exports.up = async (Knex) => {
-  await Knex.schema.createTable('movies_locations', (table) => {
+  await Knex.schema.createTable('locations_movies', (table) => {
```

Comment:
Is this change necessary? Do we know why if so?

Reply:
I think this was changed because by default, bookshelf defines the joining table name alphabetically when fetching. I believe there is a way to override this. Let me know if I should revert this to conform to the previous PR.